src/misc/setup.cpp

```diff
@@ -382,7 +384,11 @@ bool Prop_string::CheckValue(const Value &in, bool warn)
 			}
 		}
 	}
-	if (warn) LOG_MSG("\"%s\" is not a valid value for variable: %s.\nIt might now be reset to the default value: %s",in.ToString().c_str(),propname.c_str(),default_value.ToString().c_str());
+	if (warn)
+		LOG_WARNING("CONFIG: \"%s\" is an invalid value for: %s. Using the default: %s",
```

Comment:
Much better; "It might now be reset to the default value" was a terribly vague message 😅